Documentation/HowToUseWCFInContainers.md

```diff
@@ -0,0 +1,142 @@
+For the purpose of this demo we�ll go ahead and just create a default project template for WCF service app. Among other things this will generate a service and a data contract that look like the one below (IService1.cs):
+
+~~~
+using System;
+using System.Runtime.Serialization;
+using System.ServiceModel;
+
+namespace HelloWcfService
+{
+    [ServiceContract]
+    public interface IService1
+    {
+        [OperationContract]
+        string GetData(int value);
+        [OperationContract]
+        CompositeType GetDataUsingDataContract(CompositeType composite);
+    }
+
+    [DataContract]
+    public class CompositeType
+    {
+        bool boolValue = true;
+        string stringValue = "Hello ";
+        [DataMember]
+        public bool BoolValue { get { return boolValue; } set { boolValue = value; } }
+        [DataMember]
+        public string StringValue  { get { return stringValue; } set { stringValue = value; } }
+    }
+}
+~~~
+
+For further simplicity let�s skip the �standard� path of service reference generation and simply make a copy of it for our future WCF core client. Next, we�re going to write our WCF client code (program.cs):
+
+~~~
+using HelloWcfService;
+using System;
+using System.ServiceModel;
+
+namespace ConsoleApp1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var url = Environment.GetEnvironmentVariable("url");
+            var helloEndpoint = new EndpointAddress(url);
+            var binding = new BasicHttpBinding();
+            var factory = new ChannelFactory<IService1>(binding, helloEndpoint);
+            var client = factory.CreateChannel();
+            var data = client.GetData(5);
+            Console.WriteLine(data);
+        }
+    }
+}
+~~~
+
+Finally, we�ll need a project file. Unfortunately the project file may look different depending on the version of .NET core SDK you�re going to use in the container. One way to generate a project file compatible with a particular version of SDK is to run the following:
+
+~~~
+mkdir wcfClient
+cd wcfClient
+dotnet new console
+~~~
+
+(make sure the directory you�re running this in doesn�t already have a project file) and use the newly created project file as a template for your WCF client project. The last step is to add references to WCF.NET Core project like the one below. Again, depending on the SDK version, the format may change. The one that we�re providing here will be compatible with the latest 1.1.1 nightly build (wcfClient.csproj):
+
+~~~
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.App">
+      <Version>1.0.1</Version>
```

Comment:
Are you sure the version is 1.0.1? I expect the version is 1.1.1 since you have TargetFramework `netcoreapp1.1` above.

BTW, I remember in the final VS2017 build, the package reference to `Microsoft.NETCore.App` is not present anymore for projects created from default template. Can you please double check?